xbmc/cores/RetroPlayer/playback/GameLoop.h

```diff
@@ -54,15 +54,15 @@ class CGameLoop : protected CThread
   void Process() override;
 
 private:
-  double FrameTimeMs() const;
-  double SleepTimeMs() const;
-  double NowMs() const;
+  int64_t FrameTimeUs() const;
+  int64_t inline SleepTimeUs() const;
+  int64_t NowUs() const;
 
   IGameLoopCallback* const m_callback;
   const double m_fps;
   std::atomic<double> m_speedFactor;
-  double m_lastFrameMs = 0.0;
-  mutable double m_adjustTime = 0.0;
+  int64_t m_lastFrameUs = 0;
```

Comment:
If possible we should just use `std::chrono` and propagate that all the way throughout.

Reply:
Yes, std::chrono everywhere is best

Reply:
Sounds good. Updated to use `std::chrono` everywhere and re-tested.